docker-compose.yml

```diff
@@ -27,6 +27,6 @@ services:
     volumes:
       - ./tmp/onos:/root/onos/apache-karaf-4.2.6/data/tmp
```

Comment:
2.2.1 uses karaf 2.2.8 so this should be updated, otherwise logs cannot be accessed